src/auth0/handlers/guardianPhoneFactorMessageTypes.js

```diff
@@ -28,11 +42,21 @@ export default class GuardianPhoneMessageTypesHandler extends DefaultHandler {
   async getType() {
     // in case client version does not support the operation
     if (!this.client.guardian || typeof this.client.guardian.getPhoneFactorMessageTypes !== 'function') {
-      return null;
```

Comment:
`null` is not very well handled by auth0-deploy-cli@5.0.0. In order to get this change compatible with it, I'm reverting to using empty objects (like some other handlers are already doing). Not the most semantically accurate way to handle this, but we're a bit out of options if we don't want a new major bump of this lib.